.github/workflows/terraform-ci.yaml

```diff
@@ -0,0 +1,125 @@
+name: Terraform Checks
```

Comment:
for consistency, let's use the yml extension instead of yaml

Reply:
in fact that wouldn't be a problem, but all other workflows you created in this PR have yml extension. so I'd stick to yml for consistency

Reply:
Just seen that Github uses the `dependabot.yml` filename by default, so I'll use `.yml` as well for these workflows.